plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinHighLevelFunctionParameterInfoHandler.kt

```diff
@@ -298,7 +332,8 @@ abstract class KotlinHighLevelParameterInfoWithCallHandlerBase<TArgumentList : K
         if (currentArgumentIndex < 0) return false
 
         val callInfo = itemToShow.callInfo ?: return false
-        val (callElement, valueArguments, arguments, argumentMapping, valueParameterCount, parameterIndexToText, isCallResolvedToCandidate) = callInfo
+        val (callElement, valueArguments, arguments, argumentToParameterIndex, valueParameterCount, parameterIndexToText,
```

Comment:
Do we really need such big destructing declaration? It is easy to accidentally mix up some parameters. What do you think about extracting the following function body intro a separate function with `callinfo` as receiver or using `with(callInfo) { .. }` syntax here?

Reply:
Using `with(callInfo)` is a good idea, thanks for the suggestion!